web/src/lib/components/shared-components/search-bar/search-filter-modal.svelte

```diff
@@ -50,7 +54,7 @@
     return value === null ? undefined : value;
   }
 
-  let filter: SearchFilter = {
+  let filter: SearchFilter = $state({
```

Comment:
This will break the set for `personIds`, because it's updated like this:
```ts
personIds.add(id);
personIds = personIds;
```

In non-runes mode, this assignment triggers an update, but in runes mode it doesn't because the value hasn't changed. `SvelteSet` can be used as a replacement or all assignments should create a new set.